html_layout/style.css

```diff
@@ -0,0 +1,22 @@
+body {
+  background-color: #C8C8C8;
+  color: white;
+}
+
+.highlighted {
+  background-color: white;
+}
+
+
+#keyboardKeys {
```

Comment:
It's a bad idea to use id for styles, it would be better to add a class and use it for styling.